LICENSE

```diff
@@ -0,0 +1,20 @@
+MIT License
+
+Copyright (c) 2018 Coding and Community, Rensselaer Polytechnic Institute
```

Comment:
Is the copyright shared by both Coding and Community and RPI, or are you indicating that Coding and Community is part of RPI?

Reply:
TBH I didn't really think anybody was going to look that closely. I put both cause I wasn't sure if Coding and Community was a legal entity.

Maybe "Coding and Community at Rensselaer Polytechnic Institute"?